src/test/kotlin/com/github/pgreze/process/ProcessKtTest.kt

```diff
@@ -179,6 +179,30 @@ class ProcessKtTest {
         stderr shouldBeEqualTo ERR.toList()
     }
 
+    @Test
+    fun `use consume on stdout and stderr in parallel`(
+        @TempDir dir: Path,
+    ) = runSuspendTest {
+        val stdoutAndStdErr = mutableListOf<String>()
+
+        val output = process(
+            command = arrayOf(
+                "bash",
+                "-c",
+                "(echo 1 >&2; sleep 0.1; echo 2; sleep 0.1; echo 3 >&2; sleep 0.1; echo 4; sleep 0.1;)"
```

Comment:
in my local testing, 100ms is enough to make this non flaky - with 10ms you still get those weird race conditions between stdout and stderr, not sure if there is another way to avoid them like flushing them manually or something like that